app/utils/redisQueue.js

```diff
@@ -0,0 +1,71 @@
+const Queue = require('bee-queue');
+const nconf = require('nconf');
+
+const queues = {};
+
+const buildQueue = (name, params) => new Promise((resolve, reject) => {
+  const { redis: { host, port } } = params;
+  const key = host + port + name;
+
+  if (!queues[key]) {
+    const queue = new Queue(name, params);
+
+    queue.checkStalledJobs(nconf.get('redis:checkStalledJobsInterval'));
+
+    const queueInitTimeout = setTimeout(() => {
+      reject(new Error(`Redis client initialization timeout: ${key}`));
+    }, 5000);
+
+    queue.on('ready', () => {
+      clearTimeout(queueInitTimeout);
+      queues[key] = queue;
+      resolve(queue);
+    });
+  } else {
+    resolve(queues[key]);
+  }
+});
+
+const createWorkerQueue = (queueName, host = '127.0.0.1', port = 6379) => {
```

Comment:
move host and port to config

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/18/commits/f63f51034c6dc644adf3b8100c3aa5ca0cc31eae#diff-6b4ed25cf033c0ad808af3a83dd3f75cR19)